pod/perldiag.pod

```diff
@@ -7357,6 +7357,10 @@ See L<perlunicode/Wildcards in Property Values>.
 it already went past any symlink you are presumably trying to look for.
 The operation returned C<undef>.  Use a filename instead.
 
+=item Use of non-ASCII character 0x%X illegal when 'use source::encoding "ASCII"' is in effect
```

Comment:
The pragma module spells it `"ascii"` in lower-case. I don't have any strong opinion on which case it should be, but this message ought to agree with the module itself.

Reply:
I didn't mention that case doesn't matter.  I think that's the best option.  And I could mention it, or not; but if I don't, then it ought to be consistently cased, yes

Reply:
I think case should matter. Accepting extra ways to specify the same thing just makes the interface more complex and increases future design debt.

Reply:
change to make lower case required